Fixing terminal command

mojowill opened the pull request:
apt-get update doesn't accept any parameters the `-y` should be passed to the install command

marcosnils commented:
hey @jpetazzo the README has an instruction about how to spin a jekyll container to view your changes locally. That didn't work for you?

marcosnils commented:
@jpetazzo can you please add `terms: 0` to the post header?. That will make the page not to show any terminals on the bday site.

jpetazzo commented:
Ah, I feel silly, I tried to display it with my Jekyll build (the one I use for my blog) and it didn't work; I didn't see the README. :man_facepalming: :man_facepalming: :man_facepalming: 

- added `terms: 0`
- fixed display order
- shortened titles to avoid breaking the layout

✔️  ready to go!

marcosnils commented:
:tada:

mojowill committed:
Fixing terminal command

apt-get update doesn't accept any parameters the `-y` should be passed to the install command
